Modules/Loadable/Segmentations/EditorEffects/Python/SegmentEditorThresholdEffect.py

```diff
@@ -211,38 +212,51 @@ def setupOptionsFrame(self):
     histogramBrushFrame = qt.QHBoxLayout()
     histogramFrame.addLayout(histogramBrushFrame)
 
+    self.regionLabel = qt.QLabel("Region shape:")
+    histogramBrushFrame.addWidget(self.regionLabel)
+
     self.histogramBrushButtonGroup = qt.QButtonGroup()
     self.histogramBrushButtonGroup.setExclusive(True)
 
-    self.boxROIButton = qt.QPushButton()
+    self.boxROIButton = qt.QToolButton()
     self.boxROIButton.setText("Box")
     self.boxROIButton.setCheckable(True)
     self.boxROIButton.clicked.connect(self.updateMRMLFromGUI)
     histogramBrushFrame.addWidget(self.boxROIButton)
     self.histogramBrushButtonGroup.addButton(self.boxROIButton)
 
-    self.circleROIButton = qt.QPushButton()
+    self.circleROIButton = qt.QToolButton()
     self.circleROIButton.setText("Circle")
     self.circleROIButton.setCheckable(True)
     self.circleROIButton.clicked.connect(self.updateMRMLFromGUI)
     histogramBrushFrame.addWidget(self.circleROIButton)
     self.histogramBrushButtonGroup.addButton(self.circleROIButton)
 
-    self.drawROIButton = qt.QPushButton()
+    self.drawROIButton = qt.QToolButton()
     self.drawROIButton.setText("Draw")
     self.drawROIButton.setCheckable(True)
     self.drawROIButton.clicked.connect(self.updateMRMLFromGUI)
     histogramBrushFrame.addWidget(self.drawROIButton)
     self.histogramBrushButtonGroup.addButton(self.drawROIButton)
 
-    self.lineROIButton = qt.QPushButton()
+    self.lineROIButton = qt.QToolButton()
     self.lineROIButton.setText("Line")
     self.lineROIButton.setCheckable(True)
     self.lineROIButton.clicked.connect(self.updateMRMLFromGUI)
     histogramBrushFrame.addWidget(self.lineROIButton)
     self.histogramBrushButtonGroup.addButton(self.lineROIButton)
 
+    histogramBrushFrame.addStretch()
+
     self.histogramView = ctk.ctkTransferFunctionView()
+    self.histogramView.setMinimumWidth(30)
+    # qt.QSizePolicy(qt.QSizePolicy.Expanding, qt.QSizePolicy.Expanding)
+    # fails on some systems, therefore set the policies using separate method calls
+    #qSize = qt.QSizePolicy()
+    #qSize.setHorizontalPolicy(qt.QSizePolicy.Expanding)
+    #self.histogramView.setSizePolicy(qSize)
+    #self.histogramView.setMaximumWidth(50)
```

Comment:
Is this some dead code/

Reply:
Good catch, it was just testing code that I left in. I've removed it now.